lib/src/native/realm_core.dart

```diff
@@ -187,8 +187,8 @@ class _RealmCore {
 
   void deleteRealmFiles(String path) {
     using((Arena arena) {
-      final realm_deleted = arena<Uint8>();
```

Comment:
lets keep the underscore names as non idiomatic Dart names to keep treat them with care as pointers